mancala/server/src/lib.rs

```diff
@@ -28,6 +33,7 @@ fn handle_client(mut stream: TcpStream) {
                 );
                 // Close read & write portions of connection
                 stream.shutdown(Shutdown::Both).unwrap();
+                return;
```

Comment:
hold on. I committed both our solutions. Conducting further tests!

Reply:
removed the return in `Err` since checking the size of the read buffer was sufficient.